modules/porous_flow/test/tests/sinks/PorousFlowPiecewiseLinearSink_BC_eg1.i

```diff
@@ -0,0 +1,131 @@
+## This is an example input file showing how to set a Type I BC with PorousFlowPiecewiseLinearSink
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  xmin = 0
+  xmax = 1
+  ny = 10
+  ymin = 0
+  ymax = 1
+[]
+
+[MeshModifiers]
+  [./aquifer]
+    type = SubdomainBoundingBox
+    block_id = 11
+    bottom_left = '0 0 0'
+    top_right = '1 1 1'
+  [../]
+[]
+
+[GlobalParams]
+  PorousFlowDictator = dictator
+[]
+
+[Variables]
+  [./porepressure]
+    initial_condition = 1.e6 # initial pressure in domain
+  [../]
+[]
+
+[PorousFlowBasicTHM]
+  porepressure = porepressure
+  coupling_type = Hydro
+  gravity = '0 0 0'
+  fp = the_simple_fluid
+  number_fluid_phases = 1
+  number_fluid_components = 1
+[]
+
+[AuxVariables]
+  [./fluxes_out]
+  [../]
+[]
+
+[BCs]
+  [./in_left]
+    type = PorousFlowPiecewiseLinearSink
+    variable = porepressure
+    boundary = 'left'
+    pt_vals = '-1e9 1e9' # x coordinates defining g
+    multipliers = '-1e9 1e9' # y coordinates defining g
+    PT_shift = 2.E6   # BC pressure
+    flux_function = 1E-6 # Variable C
+    fluid_phase = 0
+    save_in = fluxes_out
+  [../]
+[]
+
+[Postprocessors]
+  [./left_flux]
+    type = NodalSum
+    boundary = 'left'
+    variable = fluxes_out
+    execute_on = 'timestep_end'
+  [../]
+[]
+
+[Modules]
+  [./FluidProperties]
+    [./the_simple_fluid]
+      type = SimpleFluidProperties
+      bulk_modulus = 2E9
```

Comment:
Probably you don't need to set bulk_mod, viscosity and density0 as i think you're using the default values.  Possibly you want to set thermal_expansion to zero

Reply:
Ok, sounds good.

Reply:
Viscosity retained for comparison to analytical solution.